### DISCUSSION TITLE
Maui BlazorWebView MacOS issues

### DISCUSSION THREAD

**Discussion by johngsandiford** [dotnet/maui#10847] [Q&A]
### Description

We are having some serious issues with BlazorWebView on MacOS.

Not only do we experience issues in our own code, but also in third party controls (e.g. Syncfusion grid shows duplicate columns)

Please note that we have been developing using Blazor successfully for some time, and are now using Maui for a Native  desktop experience.

We have created the following:
1. Blazor Server Side App on Windows/Mac/Linux - all works well
2. Blazor WASM online app - works well - surprisingly fast :)
3. Maui Blazor App on Windows with MSIX installer - works great
4. Maui Blazor App for Mac Desktop (MacCatalyst) - packaged with .pkg - lots of issues

We only get issues with Maui + Blazor + MacOS

We would appreciate some guidance, as we have found so many issues it is difficult to raise atomic bug reports.
We understand that some of the issues could be Mac WebView issues, but that does not really help our customers.

e.g. in Maui Blazor MacOS...

1. The top part of the app is truncated (which is unfortunately the main toolbar for us), until you maximize (green button).  If you reduce (orange button) the toolbar is again truncated.  This does not happen in 1-3 above.
2. If you load a slightly larger file (which works fine in Blazor + Chrome) the app hangs, uses lots of memory and you have to kill the app.  This does not happen in 1-3 above.
3. Third party controls such as Syncfusion grid show duplicate columns.  This does not happen in 1-3 above.

Appreciate some help!  Thanks in advance.

### Steps to Reproduce

So many issues, difficult to know where to start

### Link to public reproduction project repository

private but can provide pkg if required

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

macOS

### Affected platform versions

Latest mac

**Comment by drasticactions** [comment]
@johngsandiford I converted this to a discussion because there are several issues you wrote up, and I'm not sure if they are actionable as a single one. If you can separate these issues out as their own, that would be greatly appreciated.

**Reply by johngsandiford** [reply]
Thanks for the reply.  I have created a repro of the first issue (app truncation) at #10911
